frontends/web/src/components/status/status.tsx

```diff
@@ -89,15 +91,15 @@ export default class Status extends Component<Props, State> {
             show,
         }: State) {
         const childrenList = children as ComponentChild[];
-        if ((dismissable && !show) || (childrenList && childrenList.length === 1 && !childrenList[0])) {
+        if (!show || (childrenList && childrenList.length === 1 && !childrenList[0])) {
             return null;
         }
         return (
             <div className={[style.container, style[type], className ? className : ''].join(' ')}>
                 <div className={style.status}>
                     {children}
                     {
-                        dismissable && (
+                        dismissable !== undefined && (
```

Comment:
I'm not an expert, but can it also be `null`?

Reply:
No, that would have to be explicitly typed as `... | null`. the `?: ...` type means something or `undefined`.